changelog/11694.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
```

Comment:
We don't usually do changelog entries if there's no user-visible impact.  Since as you say these changes should be no-ops (ignoring test changes), I suggest you delete this file.

Reply:
I'll delete this changelog file, thanks!